resources/fsh-generated/resources/Patient-Example-patient-de-basis-address.json

```diff
@@ -17,29 +17,20 @@
       "type": "physical",
       "text": "Musterweg 42, Hinterhof 2. Etage, 10787 Berlin, Deutschland",
       "line": [
-        "Musterweg 42",
-        "Hinterhof 2. Etage"
+        "Musterweg 42 // Hinterhof 2. Etage"
       ],
       "_line": [
         {
           "extension": [
             {
-              "url": "http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-streetName",
```

Comment:
@alexzautke kann es sein, dass durch den MR die streetName-Extension in dem JSON verschwindet?